wallet/txauthor/author.go

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+// Package txauthor provides transaction creation code for wallets.
+package txauthor
+
+import (
+	"errors"
+
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcwallet/wallet/txfees"
+
+	h "github.com/btcsuite/btcwallet/internal/helpers"
```

Comment:
The `walletrpc` package, which is generated by `protoc`, is imported using the `pb` qualifier, so that's one example.  There may be others but that's the only one I've recently used.